.github/workflows/check-dataviews-changelog.yml

```diff
@@ -0,0 +1,79 @@
+name: OPTIONAL - Verify @wordpress/dataviews CHANGELOG update
+
+on:
+    pull_request:
+        types: [opened, synchronize]
+        branches:
+            - trunk
+        paths:
+            - 'packages/dataviews/**'
+            - '!packages/dataviews/src/**/stories/**'
+            - '!packages/dataviews/src/**/test/**'
+            - '!packages/dataviews/src/**/*.android.js'
+            - '!packages/dataviews/src/**/*.ios.js'
+            - '!packages/dataviews/src/**/*.native.js'
+            - '!packages/dataviews/src/**/*.native.scss'
+            - '!packages/dataviews/src/**/react-native-*'
+
+# Disable permissions for all available scopes by default.
+# Any needed permissions should be configured at the job level.
+permissions: {}
+
+jobs:
+    check:
+        name: Check CHANGELOG diff
+        runs-on: ubuntu-latest
+        permissions:
+            contents: read
+        steps:
+            - name: 'Get PR commit count'
+              env:
+                  PR_COUNT: ${{ github.event.pull_request.commits }}
+              run: echo "PR_COMMIT_COUNT=$(( PR_COUNT + 1 ))" >> "${GITHUB_ENV}"
+            - name: Checkout code
+              uses: actions/checkout@11bd71901bbe5b1630ceea73d27597364c9af683 # v4.2.2
+              with:
+                  ref: ${{ github.event.pull_request.head.ref }}
+                  repository: ${{ github.event.pull_request.head.repo.full_name }}
+                  fetch-depth: ${{ env.PR_COMMIT_COUNT }}
+                  show-progress: ${{ runner.debug == '1' && 'true' || 'false' }}
```

Comment:
Just a little late FYI: I _believe_ GitHub Actions will correctly cast expressions to booleans without our needing to spell it out:

```yaml
show-progress: ${{ runner.debug == '1' }}
```

if not, there's also the `fromJson` builtin:

```yaml
show-progress: ${{ fromJson( runner.debug == '1' ) }}
```

Source: https://docs.github.com/en/actions/reference/workflows-and-actions/expressions

(No action needed here, @oandregal, just an FYI for the next time we touch this stuff. I don't like these DSLs, but we ought to learn them a little more.)